### ISSUE TITLE
Visual column alignment for tables

### ISSUE THREAD

**Issue by klonuo** [rvben/rumdl#559]
### Problem or motivation

Often markdown tables are not with aligned columns, and looking at the table is painful

For example llm output:
```
| Location | Function | Parameter | Usage |
|----------|----------|-----------|-------|
| `src/source.py:120` | `fetch_measurements()` | `incremental: Optional[dlt.sources.incremental] = None` | Defined, never receives a value |
| `src/source.py:40` | `measurements_resource()` | `incremental: Optional[dlt.sources.incremental] = None` | Passes through to `fetch_measurements` |
| `src/pipeline.py:166` | `run_pipeline()` | `incremental: Optional[dlt.sources.incremental] = None` | Passes through to `measurements_resource` |
| `src/main.py:208` | — | — | Calls `run_pipeline()` **without** passing `incremental` |
```
which is not that bad (I just quickly browsed my session) and although it's correct table formatting it's almost unreadable in editor

### Proposed solution

Please allow table formatting rule to visually align table columns, maybe find greatest common factor for columns and apply to all rows:

```
| Location              | Function                  | Parameter                                               | Usage                                                    |
|-----------------------|---------------------------|---------------------------------------------------------|----------------------------------------------------------|
| `src/source.py:120`   | `fetch_measurements()`    | `incremental: Optional[dlt.sources.incremental] = None` | Defined, never receives a value                          |
| `src/source.py:40`    | `measurements_resource()` | `incremental: Optional[dlt.sources.incremental] = None` | Passes through to `fetch_measurements`                   |
| `src/pipeline.py:166` | `run_pipeline()`          | `incremental: Optional[dlt.sources.incremental] = None` | Passes through to `measurements_resource`                |
| `src/main.py:208`     | —                         | —                                                       | Calls `run_pipeline()` **without** passing `incremental` |
```

### Feature category

LSP / editor integration

### Example markdown

```markdown

```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
Thanks for the suggestion, @klonuo. This is already supported via **MD060** (table formatting). It's disabled by default, but you can enable it with the `aligned` style:

```toml
[MD060]
enabled = true
style = "aligned"
```

Then `rumdl fmt` will pad all table columns to equal width, exactly as shown in your proposed solution. It handles Unicode characters, inline code, and other edge cases.

Other available styles: `compact` (single space padding), `tight` (no padding), `aligned-no-space`.

See the [MD060 docs](https://rumdl.dev/rules/md060) for full configuration options including per-column alignment (`column-align`).
